deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1989,30 +1950,23 @@ def registry_key_cud(root_key, registry_sub_key, log_monitor, arch=KEY_WOW64_64K
 
         registry_path = os.path.join(root_key, registry_sub_key)
 
-        if not isinstance(key_list, list) and not isinstance(key_list, dict):
-            raise ValueError('Value error. It can only be list or dict')
-        elif isinstance(key_list, list):
-            aux_dict = {registry_path: ('', callback_detect_event)}
-
+        aux_dict = {}
+        if isinstance(key_list, list):
             for elem in key_list:
                 aux_dict[elem] = ('', callback)
 
-            key_list = aux_dict
         elif isinstance(key_list, dict):
-            aux_dict = {registry_path: ('', callback_detect_event)}
-
             for key, elem in key_list.items():
                 aux_dict[key] = (elem, callback)
+        else:
+            raise ValueError('Value error. It can only be list or dict')
```

Comment:
```suggestion
            raise ValueError('It can only be list or dict')
```
I think when the error is raised, it already says `ValueError`

Reply:
Done in amended commit [1f61f12](https://github.com/wazuh/wazuh-qa/pull/1670/commits/1f61f12f77f01151dd455533e24c2f23259bd755)